cob_object_detection_visualizer/ros/src/object_detection_visualizer_node.cpp

```diff
@@ -265,7 +265,7 @@ class ObjectDetectionVisualizer
 
 		cv::Mat display;
 		cv::resize(image, display, cv::Size(), 0.5, 0.5);
-		cv::imshow("object detections", display);
+        cv::imshow("object detections", display);
```

Comment:
Line 243 and 268 still have spaces instead of tabs. Please go back to the original 2x tabs indentation.

Reply:
Sorry, this line still has spaces, instead of 2 tabs.